README.md

```diff
@@ -69,6 +69,23 @@ following command:
 npm run test:jest -- -u
 ```
 
+## Deployment
+
+The production version of the Glean Dictionary
+(https://dictionary.telemetry.mozilla.org) is deployed from the `production`
+branch on this repository, which usually corresponds to the latest GitHub
+release. To update the Glean Dictionary to the latest version, follow this
```

Comment:
Is `main` the target branch when we [create a Github release](https://github.com/mozilla/glean-dictionary/releases/new)?

Reply:
Yup, I added some clarification on this.

Reply:
Thanks!